<?xml version="1.0" encoding="utf-8"?>
<launch>
  <!-- The name/namespace of this robot -->
  <arg name="name"/>
  <!-- The pointcloud topic of this robot -->
  <arg name="pc_topic" default="points"/>

  <group ns="$(arg name)">
    <!-- uav_filter node -->
    <node pkg="uav_filter" type="uav_filter" name="uav_filter">
      <remap from="input" to="$(arg pc_topic)" />
      <remap from="output" to="points_filtered_z" />
      <param name="x_max" value="0.45"/>
      <param name="x_min" value="-0.45"/>
      <param name="y_max" value="0.45"/>
      <param name="y_min" value="-0.45"/>
      <param name="z_max" value="0.3"/>
      <param name="z_min" value="-0.3"/>
    </node>
  </group>

</launch>
